content/en/security/agentless_scanning/_index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Cloud Security Management Agentless Scanning
+kind: documentation
+further_reading:
+- link: "/security/cloud_security_management/setup/agentless_scanning"
+  tag: "Documentation"
+  text: "Setting up Agentless Scanning"
+- link: "TBD"
+  tag: "Blog"
+  text: "Detect vulnerabilities in minutes with Datadog Agentless scanning for CSM"
+- link: "/security/vulnerabilities"
+  tag: "Documentation"
+  text: "Read more about CSM Vulnerabilities"
+---
+
+<div class="alert alert-info">CSM Agentless Scanning is in beta.</div>
+
+## Overview
+
+Agentless Scanning provides visibility into risks and vulnerabilities that exist within your hosts, running containers, container images, and serverless functions, without requiring you to install the Datadog Agent. Datadog recommends enabling Agentless Scanning as a first step to gain complete visibility into your resources, and install the Datadog Agent on your components over time for deeper security and observability context.
+
```

Comment:
This seems outdated based on what's in the doc: https://docs.google.com/document/d/1p8_-w_PPtMLfn4MrTDklmvHxK9oKR1Fx0XMAHuvpPlE/edit#heading=h.dua0esusvq11

i will review the Google Doc, assuming it's the final version

Reply:
Thanks! I'll work on getting this in sync :)